cmd/gaia/cli_test/cli_test.go

```diff
@@ -33,12 +33,12 @@ import (
 )
 
 func TestGaiaCLIMinimumFees(t *testing.T) {
-	gaiadHome, gaiacliHome := getTestingHomeDirs(t)
```

Comment:
It's not too bad, but perhaps these minor CLI integration test updates should be their own PR (including the `-p 4` above).